### PR TITLE
Add ModelKeeper to pub

### PR DESCRIPTION
**Pull request by dywsjtu**

### CODE REVIEW

File: source/_data/SymbioticLab.bib [hunk 1]
```diff
@@ -1205,3 +1205,39 @@ @Article{orloj:arxiv22
...
+    insight is that initializing a training job's model by transforming an already-trained model's weights can jump-start it and
+    reduce the total amount of training needed. However, models submitted over time can differ in their architectures and
+    accuracy. Given a new model to train, ModelKeeper scalably
+    identifies its architectural similarity with previously trained
+    models, selects a parent model with high similarity and good
+    model accuracy, and performs structure-aware transformation
+    of weights to preserve maximal information from the parent
+    model during the warmup of new model weights. Our evaluations across thousands of CV and NLP models show that
+    ModelKeeper achieves 1.3x-4.3x faster training completion
+    with little overhead and no reduction in model accuracy
```
Comment (jaywonchung) [inline source/_data/SymbioticLab.bib:1241]: Missing period.

File: source/_data/SymbioticLab.bib [hunk 2]
```diff
@@ -1205,3 +1205,39 @@ @Article{orloj:arxiv22
   state-of-the-art.
   }
 }
+
+@InProceedings{modelkeeper:nsdi23,
+  author    = {Fan Lai and Yinwei Dai and Harsha V. Madhyastha and Mosharaf Chowdhury},
+  booktitle = {USENIX NSDI},
+  title     = {ModelKeeper: Accelerating DNN Training via Automated Training Warmup},
```
Comment (jaywonchung) [inline source/_data/SymbioticLab.bib:1212]: Should be`{ModelKeeper}` and `{DNN}` (i.e. add brackets) to preserve capitalization. I only learned about this recently, and we probably should keep this convention for all publications at least from now on.

**Review by jaywonchung — CHANGES_REQUESTED** [review]
I did some nitpicking.

File: source/_data/SymbioticLab.bib (hunk 2 above)
Comment (dywsjtu) [reply source/_data/SymbioticLab.bib]: Thanks!

**Review by jaywonchung — APPROVED** [review]
LGTM!

**Pull request merged**